backend/src/Gitlab/Message/Async/GitlabMergeRequestUpdate.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Gitlab\Message\Async;
+
+readonly class GitlabMergeRequestUpdate
+{
+    public function __construct(
+        public string $repoURL,
+        public int $prId,
+        public string $status,
+    ) {
+    }
+}
```

Comment:
BOT4:
The file does not end with a newline. According to POSIX standards and general coding conventions, text files should end with a newline character.